### PR TITLE
move examples/websocket.rs to examples/websocket

### PR DESCRIPTION
**Pull request by AlbanMinassian**

### CODE REVIEW

File: examples/websocket/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[[bin]]
+name = "server"
+path = "src/main.rs"
+
+[dependencies]
+env_logger = "*"
+futures = "0.1"
+actix = "^0.3.5"
+actix-web = { git = "https://github.com/actix/actix-web.git", features=["signal"] }
```
Comment (fafhrd91) [inline examples/websocket/Cargo.toml:14]: I am not sure this can be build on Windows

**Review by fafhrd91 — COMMENTED** [review]
Could you also add this example to .travis

File: examples/websocket/Cargo.toml (hunk 1 above)
Comment (AlbanMinassian) [reply examples/websocket/Cargo.toml:14]: Ok i understand ... or update actix-web : if win and feature "signal" then do nothing ?
If is not possible i allow you to remove all commit of the day (I forgot to be careful).

**Review by fafhrd91 — APPROVED** [review]
lgtm

**Comment by AlbanMinassian** [comment]
I don't have windows on hand :
- http://doc.crates.io/specifying-dependencies.html#platform-specific-dependencies
- example https://github.com/blackbeam/rust-mysql-simple/blob/master/Cargo.toml

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
lgtm, please merge

**Pull request merged**